packages/react-composites/src/composites/CallComposite/styles/LocalPreview.styles.ts

```diff
@@ -59,16 +60,19 @@ export const localPreviewTileStyle = {
 /**
  * @private
  */
-export const localPreviewButtonStyle = {
-  root: {
-    '@media (forced-colors: active)': {
-      ':focus': {
-        border: '1px solid !important' // we need bang important to override fluents colours
-      },
-      ':focus::after': {
-        border: 'unset !important', // we need bang important to override fluents colours
-        outline: 'unset !important'
+export const localPreviewButtonStyle = (isMobile: boolean): IButtonStyles => {
+  return {
+    root: {
+      width: !isMobile ? _pxToRem(75) : 'unset', // we only want to enforce this for desktop views
```

Comment:
Will this work on different languages that have longer text strings? I thought we had decided that buttons could be different sizes?

Reply:
I think I am making a mistake here making it width, it should be min width so they can grow. Alex wants them to match the figmas